docs/serving/istio-authorization.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Knative application under the strict authorization policy"
+weight: 25
+type: "docs"
+---
+
+When you deployed app to Knative Serving, serving system pods such as activator and autoscaler access to your app.
+Hence, you have to allow the requests to your app when you configure security features such as istio authorization policy.
+
+> Tip: This example assumes that your application enabled istio sidecar injection.
+>
+> ```
+> $ kubectl create namespace serving-tests
+> $ kubectl label namespace serving-tests istio-injection=enabled
+> ```
+> The following policy example does not work without sidecar injection.
```

Comment:
Then why would a user use it? Is sidecar injection an option within Istio, so that we require both ways of doing this as options?